block.go

```diff
@@ -18,20 +18,23 @@
 
 package flow
 
-// A Block is a set of state mutations applied to the Flow blockchain.
+import "time"
+
+// Block is a set of state mutations applied to the Flow blockchain.
 type Block struct {
-	BlockHeader
-	BlockPayload
+	BlockHeader  *BlockHeader
```

Comment:
Sorry, I missed this in my last review. Why change these from embedded structs to fields?

Reply:
Reverted, thanks!